scripts/build_food_db.py

```
@@ -63,35 +65,37 @@ def load_source_data(self) -> tuple[List[Dict], List[Dict]]:
         else:
             print("  📊 Loading USDA from single file")
-            adapter = USDAAdapter()
+            usda_adapter = USDAAdapter()
 
         try:
-            usda_data = list(adapter.normalize())
+            usda_data = list(usda_adapter.normalize())
             print(f"  ✅ USDA: {len(usda_data)} records")
-        except Exception as e:
-            print(f"  ❌ USDA error: {e}")
+        except (OSError, ValueError, UnicodeDecodeError) as e:
```

Flake8 (7.3.0)

[error] local variable 'e' is assigned to but never used

(F841)